pentagon/migration/migrations/migration_1_2_0.py

```diff
@@ -0,0 +1,163 @@
+
+            ansible_cfg_default = '''
+[defaults]
+inventory = $INFRASTRUCTURE_REPO/plugins/inventory
+roles_path = $INFRASTRUCTURE_REPO/roles
+filter_plugins = $INFRASTRUCTURE_REPO/plugins/filter_plugins
+retry_files_save_path = ~/.ansible-retry
+hash_behavior = merge
+
+[ssh_connection]
+# this needs the path defined without the use of ENV variables
+ssh_args = -F __INFRA_REPO_PATH__/inventory/{item}/config/private/ssh_config
+'''.format(item=item)
+            self.overwrite_file('{}/config/local/ansible.cfg-default'.format(inventory_path), ansible_cfg_default)
+
+            ssh_config_default = '''
+# for the kube / kops working instances
+Host 172.20.64.* 172.20.65.* 172.20.66.* 172.20.67.* 172.20.68.* 172.20.69.* 172.20.70.* 172.20.71.* 172.20.72.* 172.20.73.* 172.20.74.* 172.20.75.*
+  User admin
+  IdentityFile __INFRA_REPO_PATH__/inventory/{item}/config/private/working_kube
+  StrictHostKeyChecking no
+  UserKnownHostsFile=/dev/null
+
+# for the kube / kops prod instances
+Host 172.20.96.* 172.20.97.* 172.20.98.* 172.20.99.* 172.20.100.* 172.20.101.* 172.20.102.* 172.20.103.* 172.20.104.* 172.20.105.* 172.20.106.* 172.20.107.*
+  User admin
+  IdentityFile __INFRA_REPO_PATH__/inventory/{item}/config/private/production_kube
+  StrictHostKeyChecking no
+  UserKnownHostsFile=/dev/null
+
+# for instances in private_working
+Host 172.20.48.* 172.20.49.* 172.20.50.* 172.20.51.* 172.20.52.* 172.20.53.* 172.20.54.* 172.20.55.* 172.20.56.* 172.20.57.* 172.20.58.* 172.20.59.*
+  User ubuntu
+  IdentityFile __INFRA_REPO_PATH__/inventory/{item}/config/private/working_private
+  StrictHostKeyChecking no
+  UserKnownHostsFile=/dev/null
+
+# for instances in private_prod
+Host 172.20.32.* 172.20.33.* 172.20.34.* 172.20.35.* 172.20.36.* 172.20.37.* 172.20.38.* 172.20.39.* 172.20.40.* 172.20.41.* 172.20.42.* 172.20.43.*
+  User ubuntu
+  IdentityFile __INFRA_REPO_PATH__/inventory/{item}/config/private/production_private
+  StrictHostKeyChecking no
+  UserKnownHostsFile=/dev/null
+
+# for instances in admin
+Host 172.20.0.* 172.20.1.* 172.20.2.* 172.20.3.* 172.20.4.* 172.20.5.* 172.20.6.* 172.20.7.* 172.20.8.* 172.20.9.* 172.20.10.* 172.20.11.*
+  User ubuntu
+  IdentityFile __INFRA_REPO_PATH__/inventory/{item}/config/private/admin_private
+  StrictHostKeyChecking no
+  UserKnownHostsFile=/dev/null
+
+# VPN instance
+# Replace the '*' with the IP address of the VPN instance
+Host *
+  User ubuntu
+  IdentityFile __INFRA_REPO_PATH__/inventory/{item}/config/private/admin_vpn
+  StrictHostKeyChecking no
+  UserKnownHostsFile=/dev/null
+'''.format(item=item)
+        self.overwrite_file('{}/config/local/ssh_config-default'.format(inventory_path), ssh_config_default)
+        # update core
+
+        self.delete('{}/config/local/ansible.cfg'.format(inventory_path))
+        self.delete('{}/config/local/ssh_config'.format(inventory_path))
+        self.delete('tests')
+        self.delete('tasks')
+        self.delete('components')
```

Comment:
With all these deletes here, is there a way to guarantee that these files have already been copied elsewhere? Or are these files just irrelevant now?

Reply:
All of these deletes are for files that should not exist in the repo at all anymore.